File: 2nd_quarter/week08/Whaeun/소수 찾기.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

bool isPrime(int n){
    for(int i = 2; i * i <= n; i++){
        if(n % i == 0){
            return false;
        }
    }
    return true;
}

int solution(int n) {
    int answer = 0;
    
    for(int i = 2; i  <= n; i++){
        if(isPrime(i)){
            answer++;
        }
    }
    
    return answer;
}

File: 2nd_quarter/week08/Rash/소수찾기.py
import math
def solution(n):
    answer = 0
    for i in range(2,n + 1):
        answer += 소수판별(i)
    return answer


def 소수판별(x):
    for i in range (2, int(math.sqrt(x) + 1)):
        if x % i == 0:
            return 0
    return 1